packages/next/src/lib/metadata/default-metadata.tsx

```diff
@@ -1,10 +1,27 @@
-import type { ResolvedMetadata } from './types/metadata-interface'
+import type {
+  ResolvedMetadata,
+  ResolvedScreenMetadata,
+} from './types/metadata-interface'
+
+export function createDefaultScreenMetadata(): ResolvedScreenMetadata {
+  return {
+    viewport: {
+      width: 'device-width',
+      initialScale: 1,
+    },
+    themeColor: null,
+    colorScheme: null,
+  }
+}
 
 export function createDefaultMetadata(): ResolvedMetadata {
   return {
-    viewport: 'width=device-width, initial-scale=1',
-    metadataBase: null,
+    // Deprecated ones
+    viewport: null,
+    themeColor: null,
+    colorScheme: null,
```

Comment:
do these need to be part of the default object? I think we should deprecate these keys on the `Metadata` type but shouldn't they just not be found at all on `ResolvedMetadata`?

Reply:
This is similar as deprecation note for types compatibility purpose, will remove them once full dropped later.